### ISSUE TITLE
Hourly Widget Request

### ISSUE THREAD

**Issue by neilromig** [martykan/forecastie#334]
I like the app but really want a widget like the existing 4x1, showing weather symbols for next few hours ahead rather than daily, perhaps choice of interval as well?

**Comment by robinpaulson** [comment]
Yes, this is worthwhile but unfortunately low priority.

**Comment by robinpaulson** [comment]
@neilromig Could you mock up a design please? Even a sketch on paper scanned or photographed and uploaded would help; I realise I can't exactly picture what you're after

**Comment by neilromig** [comment]
I used to use an app "UK Weather" which I think is not now published. It
was a version of "El Temps" by lujop. I have attached a screenshot and the
widget I like is third down showing today's weather using forecast symbols
for current time and the next few hours.

**Comment by robinpaulson** [comment]
I'm afraid I can't see the image, could you give it another go? Or point me to a website?

**Comment by neilromig** [comment]
Ok, try
www.vectorstock.com/royalty-free-vector/weather-widget-ui-set-of-the-flat-design-trend-vector-1576079
but
instead of daily interval, show hourly or two hourly.

Neil.

**Comment by FridoDeluxe** [comment]
One remark: We only have acces to a 3-hourly forecast at OWM. I'm not sure if interpolation between these 3 hour intervals is a valid approach for weather data.

**Comment by robinpaulson** [comment]
Correct, I won't be changing the forecast interval to 1hr, whether by upgrading our access to OWM, or interpolating.
The widget layout is nice though, it's worth considering having a wdiget which shows the future.

**Comment by robinpaulson** [comment]
This is almost identical to #158 , but in widget form

**Comment by neilromig** [comment]
I will keep my fingers crossed someone wants to code it! A three hour interval would be fine, it's just a way of viewing the expected weather for the rest of the day. I agree with the sentiments of #158.
